### PR TITLE
SAK-47071 Calculated Questions - Add spaces between answers

### PR DESCRIPTION
**Pull request by feralvarez95**
https://sakaiproject.atlassian.net/browse/SAK-47071

### CODE REVIEW

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/delivery/DeliveryActionListener.java [hunk 1]
```diff
@@ -2911,11 +2911,11 @@ private String commaDelimitedCalcQuestionAnswers(ItemDataIfc item, DeliveryBean
 		  // We need the key formatted in scientificNotation
 		  answer = service.toScientificNotation(answer, decimalPlaces);
 		  
-		  keysString = keysString.concat(answer + ",");
+		  keysString = keysString.concat(answer + ", ");
 		  answerSequence++;
 	  }
 	  if (keysString.length() > 2) {
```
Comment (jesusmmp) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/delivery/DeliveryActionListener.java:2917]: Should be this > 3?

Comment (feralvarez95) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/delivery/DeliveryActionListener.java:2917]: That was a bug, for a single answer with an integer of a single digit a comma was been shown. Currently in nightly looks this way:
image

**Comment by feralvarez95** [comment]
After changes answer will be shown as you can see in the next picture
image
And in spanish looks this way
image

**Comment by kunaljaykam** [comment]
This commit takes care of this issue too: https://sakaiproject.atlassian.net/browse/SAK-43626

👍

**Comment by jesusmmp** [comment]
> This commit takes care of this issue too: https://sakaiproject.atlassian.net/browse/SAK-43626
> 
> 👍

Yes @kunaljaykam. This patch solves that too. :)

**Review by juanjmerono — APPROVED** [review]

**Pull request merged**